nixos/modules/services/backup/borgbackup.nix

```
@@ -555,6 +553,19 @@ in {
             example = "--save-space";
           };
 
+          extraPackages = mkOption {
+            type = with types; listOf package;
+            default = [];
+            defaultText = literalExample "[];";
```

```suggestion
            defaultText = literalExample "[]";
```
but this shouldn't be necessary

---

I'll add this.

---

I think you can remove it completely.